pkg/observer/data.go

```diff
@@ -45,7 +45,7 @@ func add(r *bytes.Reader, m *dataapi.MsgData) error {
 		dataMap.Add(m.Id, data)
 	}
 
-	logger.GetLogger().Debugf("Data message received id %v, size %v, total %v", m.Id, size, len(data))
+	logger.GetLogger().WithFields(nil).Tracef("Data message received id %v, size %v, total %v", m.Id, size, len(data))
```

Comment:
I guess the `WithFields` can be removed? Or is the idea to encode the fields there?